xcode/Subconscious/Shared/Components/Notebook/Notebook.swift

```diff
@@ -938,7 +925,22 @@ struct NotebookModel: ModelProtocol {
                     environment: environment
                 )
             }
-            
+        }
+        
+        // If slashlink pointing to our sphere, dispatch findAndPushEditDetail
+        // to find in local or sphere content and then push editor detail.
+        guard slashlink.peer != nil else {
+            return update(
+                state: state,
+                action: .findAndPushMemoEditorDetail(
+                    slug: slashlink.toSlug(),
+                    fallback: fallback
+                ),
+                environment: environment
+            )
+        }
```

Comment:
This needed to move down _after_ checking for a profile link, we were short circuiting when finding local content but profile's should not be caught by that.